packages/pigeon/pigeons/core_tests.dart

```diff
@@ -50,23 +116,13 @@ abstract class HostIntegrationCoreApi {
   @ObjCSelector('echoAllTypes:')
   AllTypes echoAllTypes(AllTypes everything);
 
+  /// Returns the passed object, to test serialization and deserialization.
+  @ObjCSelector('echoAllNullableTypes:')
+  AllNullableTypes echoAllNullableTypes(AllNullableTypes everything);
```

Comment:
I would make the argument and return value nullable here to add more coverage; it can then be used to test both nullable objects themselves, as well as nullable fields.